ui/src/scss/_sizing.scss

```diff
@@ -186,6 +186,9 @@ $abbrev in (width: w, height: h) {
 .min-w-250p {
   min-width: 250px;
 }
+.h-100-n60{
```

Comment:
what does it mean n-60 and it seems, the class is nowhere used @manikanta-reddy-t

Reply:
n-60 means negative 60

Reply:
used in latest commit,forgot to add in earlier commit